structurizr/workspace.dsl

```diff
@@ -0,0 +1,260 @@
+				tags "LegacySystem"
+			}
+
+            systemLegacyBo = softwareSystem "Horizon" "Legacy internal service to manage planning appeals in England" {
+				tags "LegacySystem"
+			}
+
+			# Relationships
+
+			// Deployment
+			containerPipelines -> systemGithub "Retrieves code"
+			containerPipelines -> containerKeyVault "Retrieves secrets"
+			containerPipelines -> containerRegistry "Pushes new container versions to registry"
+			systemAppsFo -> containerRegistry "Apps pull latest container version"
+			containerInfraPipelines -> systemAppsFo "Terraform updates infra"
+
+			// users
+			userCaseOfficer -> containerBoWeb "Manages cases"
+			userInspector -> containerBoWeb "Decides cases"
+
+			userAppellant ->  containerFrontDoor "Registers appeals"
+            userLocalPlanningAuthority ->  containerFrontDoor "Responds to appeals"
+            userInterested -> containerFrontDoor "Comments on appeals"
+            userRule6 -> containerFrontDoor "Comments on appeals"
+
+			// Networking
+			containerFrontDoor -> containerAppealsWAF "Check traffic with WAF"
+			containerFrontDoor -> containerFoWeb "Forward valid traffic to user facing website"
+			systemAppsFo -> systemAppsBo "Peered VNets"
+
+			// Forms Web App
+            containerFoWeb -> containerFoApi "gets/sets appeal data"
+            containerFoWeb -> containerFoDocsApi "gets/sets documents"
+            containerFoWeb -> containerFoAuthServer "gets auth tokens"
+            containerFoWeb -> containerFoPdfApi "creates pdfs"
+			containerFoWeb -> containerFoClamAV "Scans file uploads"
+            containerFoWeb -> containerFoAzureCosmos "stores user session data"
+			containerFoWeb -> containerKeyVault "retrieves secrets"
+            
+			// Appeals API
+            containerFoApi -> containerFoAzureSql "stores data"
+            containerFoApi -> containerFoAzureCosmos "stores data"
+            containerFoApi -> containerFoAuthServer "validates auth tokens"
+            containerFoApi -> componentCaseCmdTopic "informs back office of new appeals"
+            containerFoApi -> componentLpaqCmdTopic "informs back office of new lpaqs"
+			containerFoApi -> containerKeyVault "retrieves secrets"
+
+			// Docs API
+            containerFoDocsApi -> containerFoAzureSql "stores data"
+            containerFoDocsApi -> containerFoAzureCosmos "stores data"
+            containerFoDocsApi -> containerFoAuthServer "validates auth tokens"
+            containerFoDocsApi -> containerFoFileStorage "stores/retrieves documents"
+            containerFoDocsApi -> containerBoFileStorage "retrives BO docs"
+			containerFoDocsApi -> containerKeyVault "retrieves secrets"
+
+
+			// Notify
+            containerFoApi -> containerGovNotify "Sends emails"
+            containerFoAuthServer -> containerGovNotify "Sends emails"
+			containerBoWeb -> containerGovNotify "Sends emails"
+			containerGovNotify -> userLocalPlanningAuthority "Notifies the LPA"
```

Comment:
With relationships you can specify a description, technology and tags (https://docs.structurizr.com/dsl/language#relationship)

I'd suggest considering being more explicit in these cases and mention the how the persona/person is being notified, whilst also including that as the technology. As there's are several different ways that GOV Notify can send notifications

i.e. `containerGovNotify -> userLocalPlanningAuthority "Emails the LPA" "Email/SMTP`

Reply:
Have added detail where I know it and hopefully made more explicit